lib/model/model.go

```diff
@@ -2893,3 +2900,17 @@ func (s folderDeviceSet) hasDevice(dev protocol.DeviceID) bool {
 	}
 	return false
 }
+
+func ignoredFoldersMissing(from, to []config.ObservedFolder) []config.ObservedFolder {
```

Comment:
This function would be unnecessary if ignored folders were saved in the device config.

Reply:
And imagine all the places where we'd have to `if folder.Ignored`...

Reply:
If we save ignored folders on the device, which I think *might* be a good idea but don't feel strongly about, it should surely not be as regular folders with an "ignored" attribute, but as a specific list of ignored folders.